editor/gl/program.ts

```diff
@@ -16,6 +16,13 @@ export class Program {
         let vertex_shader = gl.createShader(gl.VERTEX_SHADER);
         let fragment_shader = gl.createShader(gl.FRAGMENT_SHADER);
         let program = gl.createProgram();
+
+        // Make sure program is not null
+        // Removing that check make the compilation fail \o/
```

Comment:
It is because tsc is smart enough, it use the control flow to compute that after the `if (program) { program = ... }`, `program` can't be null despite the fact that its type was `WebGLProgram | undefined` at the initialization point (`gl.createProgram` might return null).